### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by GurevichVarvara**

### CODE REVIEW

File: gameMuster/tests/test_views.py [hunk 1]
```diff
@@ -16,9 +48,72 @@ def test_index_get(self):
...
+        """If filters are selected by user"""
+        response = self.client.get(
+            reverse('index'),
+            {'platforms': [self.platform_1.id],
+             'genres': [self.genre_1.id]}
+        )
+
+        self.assertCountEqual(
+            [g.id for g in response.context['game_list']],
+            [self.game.id]
```
Comment (GurevichVarvara) [inline gameMuster/tests/test_views.py:91]: If I compared lists of Game objects, I'd get such error: '<Game object>' != <Game object>
That's why I decided to compare lists of ids.

Comment (GurevichVarvara) [reply gameMuster/tests/test_views.py:91]: I solved it. List comparison works correct with assertCountEqual, but not with assertQuerysetEqual.

File: gameMuster/tests/test_views.py [hunk 2]
```diff
@@ -16,9 +48,72 @@ def test_index_get(self):
...
+    def test_no_filters_selected(self):
+        """If no filters are selected by user"""
+        response = self.client.get(reverse('index'))
 
-        self.assertEquals(response.status_code, 200)
-        self.assertTemplateUsed(response, 'gameMuster/detail.html')
+        self.assertCountEqual(
+            [g.id for g in response.context['game_list']],
+            [self.game.id]
+        )
```
Comment (aleksey-kuchik) [inline gameMuster/tests/test_views.py]: You should check not the count but rather games here since count may be the same with other games, right?

**Review by aleksey-kuchik — CHANGES_REQUESTED** [review]

**Comment by GurevichVarvara** [comment]
I added model factories and removed protected method tests

File: gameMuster/tests/test_views.py (hunk 2 above)
Comment (GurevichVarvara) [reply gameMuster/tests/test_views.py]: assertCountEqual (Python 3) is the analogue of the ListItemsEqual (Python 2).

**Pull request merged**